app/adapters/registration.ts

```diff
@@ -1,6 +1,10 @@
 import OsfAdapter from './osf-adapter';
 
-export default class Registration extends OsfAdapter {}
+export default class Registration extends OsfAdapter {
+    pathForType() {
+        return 'registrations';
```

Comment:
So, this shouldn't be necessary. I did some digging and the problem is actually due to our use of ES6 classes to define models. It seems to confuse Ember and it thinks the `modelName` of a registration is `node` because it extends `Node`. This can be worked around by adding and empty `.extend()` to the registration model, e.g.
```
export default class Registration extends Node.extend() {
```